src/wallet/test/wallet_tests.cpp

```diff
@@ -812,7 +812,7 @@ BOOST_FIXTURE_TEST_CASE(ZapSelectTx, TestChain100Setup)
 
         LOCK(wallet->cs_wallet);
         BOOST_CHECK(wallet->HasWalletSpend(prev_hash));
-        BOOST_CHECK_EQUAL(wallet->mapWallet.count(block_hash), 1);
+        BOOST_CHECK_EQUAL(wallet->mapWallet.count(block_hash), (size_t)1);
```

Comment:
```suggestion
        BOOST_CHECK_EQUAL(wallet->mapWallet.count(block_hash), 1U);
```

nit: I think the other tests use the postfix thing

Reply:
> nit: I think the other tests use the postfix thing

Thanks, switched